src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -62,6 +62,13 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('no_push', help="Indicates whether the image built should be pushed to the registry.", arg_type=get_three_state_flag())
         c.argument('no_logs', help="Do not show logs after successfully queuing the build.", action='store_true')
 
+    with self.argument_context('acr import') as c:
+        c.argument('source', help="Either a fully qualified or a partially qualified source identifier. For example, 'repository:tag' or 'repository@sha' for a partially qualified image and 'registry.azurecr.io/repository:tag' or 'registry.azurecr.io/repository@sha' for a fully qualified iamge.")
+        c.argument('resource_id', options_list=['--registry', '-r'], help='The container registry of the source. It can be name, canonical name(login server) or resource id of the source registry.')
```

Comment:
Consider renaming `resource_id` to, for example, `source_registry` since you are also taking name and login server in addition to source registry ID.

Reply:
Done